scripts/install-operator-results-api.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+set -e
+
+# This script assumes that you are connected to an OpenShift cluster and have the Pipelines Operator installed
+
+echo "Installing the Pipelines Operator ... "
+
+# Install the Pipelines Operator from subscription
+cat << EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  labels:
+    operators.coreos.com/openshift-pipelines-operator-rh.openshift-operators: ""
+  name: openshift-pipelines-operator-rh
+  namespace: openshift-operators
+spec:
+  channel: latest
+  config:
+    env:
+      - name: IMAGE_RESULTS_API
+        value: quay.io/avinkuma/api-b1b7ffa9ba32f7c3020c3b68830b30a8@sha256:e1b184582f9c42fa5493edab03a76ae3adc61359217728028bf11502e144dfd2
```

Comment:
Update this value to `quay.io/avinkuma/api-b1b7ffa9ba32f7c3020c3b68830b30a8:latest`